lib/game.dart

```diff
@@ -74,6 +106,40 @@ abstract class BaseGame extends Game {
   /// List of deltas used in debug mode to calculate FPS
   final List<double> _dts = [];
 
+  bool _checkTapOverlap(Tapeable c, Offset o) {
+    final pointRect = Rect.fromLTWH(o.dx, o.dy, 1, 1);
+
+    return c.toRect().overlaps(pointRect);
+  }
+
+  Iterable<Tapeable> get _tapeableComponents =>
+      components.where((c) => c is Tapeable).cast<Tapeable>();
```

Comment:
just a question: `.cast()` won't work? AFAICT it should infer in this case (please test, I'm not sure)

Reply:
I am using `cast` there, it is working

Reply:
I mean cast without the explicit generic type specification